source/development/coredump.rst

```diff
@@ -0,0 +1,203 @@
+.. Copyright (C) 2024, Wazuh, Inc.
+
+.. meta::
+  :description: This section contains instructions to configure and collect coredumps for furture analisys
+
+.. _coredumps:
+
+HowTo configure Wazuh agent's OS to generate `coredumps`
+========================================================
+This section contains instructions to customize the diferent OS where Wazuh agent is installed.
+
+Introduction
+------------
+A **core dump** a.k.a **crash-dump** is a snapshot of the memory of a running process. The S.O. where the Wazuh agent resides, can automatically create a core dump when a serious or unhandled error occurs. Sometimes a crash dump is useful for diagnosing a process that appears to be hanging; The memory dump may reveal information about the cause of the crash.
+When collecting a core dump, be sure to collect other information about the environment so that the core file can be analyzed (for example, the agent and OS version along with any other data you consider appropriate).
+
+Linux agent's OS
+----------------
+Since Linux version 2.41 onwards, a template is implemented to define the location and name of the coredumps that are generated in the system, here the reference_.
+Unlike the previous versions, which, if enabled, generated the core file next to the location of the file that caused the error.
+
+.. _reference: https://man7.org/linux/man-pages/man5/core.5.html
+
+Using `ulimit`, without `systemd`
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
```suggestion
Manual configuration
^^^^^^^^^^^^^^^^^^^^
```

Reply:
What about bringing the "Using systemd" sub-section up before this section? Isn't the "using systemd" the usual first recommendation to handle this configuration? Maybe we could introduce it first and then the manual ulimit way.